g/grpc-gateway/grpc-gateway_ubi_8.4.sh

```diff
@@ -0,0 +1,72 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : grpc-gateway 
+# Version       : v2.6.0
```

Comment:
Is it the only version tested here ?

Reply:
@gururajrkatti As discussed in meeting I have verified v1.4.1 its not working. On master i.e. v2.6.0 version the build and tests passed.